packages/utilities/gel-foundations/src/typography.js

```diff
@@ -1,11 +1,76 @@
-import { MEDIA_QUERY_TYPOGRAPHY } from './breakpoints';
+import { MEDIA_QUERY_TYPOGRAPHY } from '../breakpoints';
 
 const fontFamilyBase = ', Helvetica, Arial, sans-serif';
 
 export const GEL_FF_REITH_SERIF = `ReithSerif${fontFamilyBase}`;
 export const GEL_FF_REITH_SANS = `ReithSans${fontFamilyBase}`;
 export const GEL_FF_REITH_SANS_COND = `ReithSansCondensed${fontFamilyBase}`;
 
+function getTypeSizes(typogStyleName, importedScript) {
```

Comment:
```suggestion
const getTypeSizes = (typogStyleName, importedScript) => {
```

Reply:
Also, could this function be moved into a new folder? and unit tested

Reply:
Agreed, this function (and inherited functions) lends itself well to being in a separate file and unit tested.

Reply:
Already moved it into a separate file. Missing unit test.